backend/internal/storage/postgres/schema/review.go

```diff
@@ -67,6 +67,28 @@ func (r *ReviewRepository) CreateReview(ctx context.Context, review *models.Revi
 		return nil, err
 	}
 
+	// Fetch or create tag IDs and add them to the review_tags table
+	for _, label := range review.Tags {
+		var tagID int
+
+		// Check if the tag already exists
+		tagQuery := `SELECT id FROM tag WHERE name = $1`
+		err := r.QueryRow(ctx, tagQuery, label).Scan(&tagID)
+
+		// If the tag doesn't exist, insert it and retrieve its ID
```

Comment:
we should prob delete/update this comment since it doesn't match the actual behavior